pkg/collector/corechecks/snmp/metadata/store.go

```diff
@@ -0,0 +1,126 @@
+package metadata
```

Comment:
Could we unit test the methods in this file?

Reply:
Sure, tested here 52aaa581d